File: IdeaProjects/MyBatisLearning/mybatis-series/chat02/src/main/java/com/javacode2018/chat02/UserMapper.java
package com.javacode2018.chat02;

import org.apache.ibatis.annotations.Mapper;

import java.util.List;

public interface UserMapper {
    int insertUser(UserModel userModel);
    int updateUser(UserModel userModel);
    int deleteUser(UserModel userModel);
    List<UserModel> getUserList();
}

File: IdeaProjects/MyBatisLearning/mybatis-series/chat04/src/main/java/com/javacode2018/chat04/demo1/mapper/UserMapper.kt
package com.javacode2018.chat04.demo1.mapper

import com.javacode2018.chat04.demo1.model.UserModel

interface UserMapper {
    fun insertUser(user:UserModel):Int
    fun updateUser(user:UserModel):Long
    fun deleteUser(user:UserModel):Boolean
    fun insertUser1(user:UserModel):Int
    fun insertUser2(user:UserModel):Int
}